src/brevitas_examples/llm/main.py

```diff
@@ -196,18 +199,34 @@ def main(args):
         remove_hooks(model)
         print(f"Float perplexity ({args.dataset}): {float_ppl:.3f}")
 
+    if args.replace_rmsnorm:
+        model = replace_rmsnorm_with_torch(model, model.config)
+
     if require_fx:
-        model = get_fx(model)
+        with torch.no_grad():
+            model, guards = torch._dynamo.export(model)(**calibration_loader[0])
```

Comment:
This has an impact on how dataloaders are created, we don't need extra kwargs for attention_mask